snykTask/src/__tests__/task-lib.test.ts

```diff
@@ -20,6 +20,10 @@ import * as fs from 'fs';
 import * as os from 'os';
 import * as path from 'path';
 import * as fse from 'fs-extra';
+import { generateSnykCodeResultsWithoutIssues } from '../index';
+import { getSnykDownloadInfo, downloadExecutable } from '../install';
+
+jest.setTimeout(120_000);
```

Comment:
**question:** What did we introduce here that made a 2 min time out required?

Reply:
The download of the CLI in beforeAll()